src/api/che.ts

```diff
@@ -39,7 +39,7 @@ export class CheHelper {
 
     const res = await k8sApi.listNamespacedPod(namespace)
     const pods = res.body.items
-    const wsPods = pods.filter(pod => pod.metadata!.labels!['che.workspace_id'])
+    const wsPods = pods.filter(pod => pod.metadata!.labels!['che.original_name'] === 'che-workspace-pod')
```

Comment:
we have these labels with all installs ? operator,helm and minishift ?

Reply:
@benoitf good question )
@sleshchenko do you happen to know?

Reply:
hum in fact it's ok, as it's che server that is creating these workspaces so we don't care about install part

Reply:
I doubt that we can rely on `che-workspace-pod` value...
As you can see - it's used only in PluginBroker component related class https://github.com/eclipse/che/search?q=che-workspace-pod&unscoped_q=che-workspace-pod
And only if environment do not have already some pod https://github.com/eclipse/che/blob/dd47af60fdc569b4770659514c0c88b0f26184a3/infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/KubernetesPluginsToolingApplier.java#L106

I assume it may not work if you use devfile with k8s/OS/dockerimage component